internal/configs/module_call.go

```diff
@@ -202,6 +148,93 @@ func decodeModuleBlock(block *hcl.Block, override bool) (*ModuleCall, hcl.Diagno
 	return mc, diags
 }
 
+func (mc *ModuleCall) decodeStaticFields(eval *StaticEvaluator) hcl.Diagnostics {
+	if mc.Source == nil {
+		// Broken, will error elsewhere
+		return nil
+	}
```

Comment:
This is a scary comment. Where will this error? Is this covered with a test?

Reply:
I found this in an existing test, I should link to that scenario in the module installer as that's where that class of errors is expected.